internal/breaking-changes.go

```diff
@@ -12,12 +12,12 @@ import (
 func handleBreakingChanges(stdout io.Writer, diffReport *diff.Diff, operationsSources *diff.OperationsSourcesMap, inputFlags *InputFlags) (bool, *ReturnError) {
 	var c checker.BackwardCompatibilityCheckConfig
 	var level checker.Level
+
+	c = checker.GetAllChecks(inputFlags.includeChecks)
```

Comment:
from now on, all checkers will be loaded, but every optional checker will default to `INFO`logs, so they will be filtered out.